02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
-# Introducing survey data {#c02}
+# Overview of Surveys {#c02}
 
+Creating and fielding surveys to provide estimates of a population are much more complex than simply adding a few questions to an online platform. Survey researchers can spend months, or even years, developing the study design, questions, and other methods for a single survey to ensure high quality data is collected. While this book focuses on the analysis methods of surveys, understanding the entire survey process can provide a better insight into what types of analyses should be conducted on the data. To gain a deeper understanding of survey design and implementation, there are many pieces of existing literature that we recommend reviewing in detail (e.g., {cite: Dillman book, Groves book, education book?, Tourangeau book, Asking Questions book?, others?}).
+
+The survey life cycle starts with a research topic or question of interest (e.g., what impact does childhood trauma have on health outcomes later in life). Researchers typically review existing data sources to determine if data are already available that can answer this question, as this can result in reduced burden on respondents, cheaper research costs, and faster research outcomes. However, if existing data cannot answer the nuances of the research question, a survey can be used to capture the exact data that the researcher needs.
+
+When starting a survey, there are multiple things to consider including the study population, how the sample is selected, and the way the question is worded. However, each step and decision can impact how the results can be interpreted, specifically related to the types of error that can be introduced into the data. This chapter provides a brief introduction to the types of error to consider when conducting surveys and how these error sources can inform the interpretations of the data.
+
+## Error Sources {#overview-error}
+
+Generally, survey researchers consider there to be 7 main sources of error ({Cite Groves book}):
+
+-  Representation
+  -  **Coverage Error**: A mismatch between the target population and the sample frame available
+  -  **Sampling Error**: Error produced when selecting a sample from the sample frame (none if conducting a census)
+  -  **Nonresponse Error**: Differences between those who responded and did not respond to the survey (unit nonresponse) or to a given question (item nonresponse)
+  -  **Adjustment Error**: Error introduced during post-survey statistical adjustments
+-  Measurement
+  -  **Validity**: A mismatch between the topic of interest and the question used to collect that information
+  -  **Measurement Error**: A mismatch between what the researcher asked and how the respondent answered
+  -  **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjusts data based on illogical responses)
```

Comment:
Need to two tabs to get the second level of the list to look how I think we want it.
```suggestion
-  Representation
    -  **Coverage Error**: A mismatch between the target population and the sample frame available
    -  **Sampling Error**: Error produced when selecting a sample from the sample frame (none if conducting a census)
    -  **Nonresponse Error**: Differences between those who responded and did not respond to the survey (unit nonresponse) or to a given question (item nonresponse)
    -  **Adjustment Error**: Error introduced during post-survey statistical adjustments
-  Measurement
    -  **Validity**: A mismatch between the topic of interest and the question used to collect that information
    -  **Measurement Error**: A mismatch between what the researcher asked and how the respondent answered
    -  **Processing Error**: Edits by the researcher to responses provided by the respondent (e.g., adjusts data based on illogical responses)
```